### PR TITLE
tidb-in-kubernetes/get-started: Create deploy-tidb-from-kubernetes-kind.md

### PR DESCRIPTION
**Pull request by Yisaer**
### What is changed, added or deleted? 

add introduction for tidb-operator quick start by kind.

### Which version does your change affect? 
v3.0, dev

### CODE REVIEW

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+    >
+    > 不同版本 `kubectl` 输出可能略有不同。
+
+- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/): 版本 >= 0.5.1
+
+## 第 1 步: 通过 kind 部署 Kubernetes 集群
+
+首先，请确认 Docker 进程正常运行。然后你可以通过 kind 命令快速启动一个本地的单点 Kubernetes 集群( kind 0.5.1 默认为 1.15.3 版本）。
+
+创建集群,这里我们先手动指定为 1.14.3 版本:
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]: Why use 1.14.3? If we recommend users to try 1.14, maybe the latest release of 1.14, e.g. 1.14.6?

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+
+本文介绍了如何在个人电脑 （Linux 或 MacOS) 上采用 [kind](https://kind.sigs.k8s.io/) 方式在 Kubernetes 上部署 [TiDB Operator](https://github.com/pingcap/tidb-operator) 和 TiDB 集群。
+
+kind 通过 Docker 容器模拟出一个单点的 Kubernetes 集群。 kind 的设计初衷是为了在本地进行 Kubernetes 集群的一致性测试，这意味着你可以使用 kind 模拟出你想要的 Kubernetes 版本集群，你可以在 [Docker hub](https://hub.docker.com/r/kindest/node/tags) 中找到你想要部署的 Kubernetes 版本。
+
+> **警告：**
+>
+> 对于生产环境，不要使用此方式进行部署。
+
+## 环境准备
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]: "net.ipv4.ip_forward = 1" should be set on host.

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+创建集群,这里我们先手动指定为 1.14.3 版本:
+
+> **注意：**
+>
+> 了解使用 kind 的更多细节，请参考 [kind 官方文档](https://kind.sigs.k8s.io/docs/user/quick-start/)
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kind create cluster --image kindest/node:v1.14.3
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]: No need to use kind-cluster-build.sh? The script also mounts dirs, installs local volume provisioner, tiller and creates tidb-cluster crd.

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+
+```
+NAME                              DATA   AGE
+demo-monitor                      5      1m
+demo-monitor-dashboard-extra-v3   2      1m
+demo-monitor-dashboard-v2         5      1m
+demo-monitor-dashboard-v3         5      1m
+demo-pd                           2      1m
+demo-tidb                         2      1m
+demo-tikv                         2      1m
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]: From tidb-cluster chart v1.0.0, configmap rollout feature is enabled by default, there should be cm with hash values for pd, tidb and tikv.

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 5]
```diff
@@ -0,0 +1,300 @@
...
+创建集群,这里我们先手动指定为 1.14.6 版本:
+
+> **注意：**
+>
+> 了解使用 kind 的更多细节，请参考 [kind 官方文档](https://kind.sigs.k8s.io/docs/user/quick-start/)
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kind create cluster --image kindest/node:v1.14.6
```
Comment (weekface) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]: 这里不是用我们项目里的 `hack/kind-cluster-build.sh` 来构建 kind 集群吗？

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 6]
```diff
@@ -0,0 +1,300 @@
...
+    $ kubectl get storageClass
+    NAME                 PROVISIONER               AGE
+    standard (default)   kubernetes.io/host-path   20h
+    ```
+
+    > **注意：**
+    >
+    > `<storageClass>` 在后续文档中代表当前 Kubernetes 集群的 StorageClass Name，例如 `standard` 。
+
+3. 安装 TiDB Operator:
```
Comment (weekface) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]: 这些步骤跟官网文档重复了：https://pingcap.com/docs-cn/v3.0/tidb-in-kubernetes/deploy/tidb-operator/#%E5%AE%89%E8%A3%85-tidb-operator

建议给个链接即可。

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 7]
```diff
@@ -0,0 +1,300 @@
...
+
+    输出类似如下内容：
+
+    ```
+    NAME                                       READY     STATUS    RESTARTS   AGE
+    tidb-controller-manager-5cd94748c7-jlvfs   1/1       Running   0          1m
+    tidb-scheduler-56757c896c-clzdg            2/2       Running   0          1m
+    ```
+
+## 第 3 步: 在 kind Kubernetes 集群中部署 TiDB 集群
```
Comment (weekface) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]: ditto

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+
+``` shell
+NAME                 PROVISIONER                    AGE
+local-storage        kubernetes.io/no-provisioner   7m50s
+standard (default)   kubernetes.io/host-path        8m29s
+```
+
+## 第 2 步: 在 kind Kubernetes 集群上部署 TiDB Operator
+
+参考[部署 TiDB Operator](https://pingcap.com/docs-cn/v3.0/tidb-in-kubernetes/deploy/tidb-operator/#%E5%AE%89%E8%A3%85-tidb-operator)
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]: Please refer to the link format in other documents, no need to set the complete path of the doc in the same repo.

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+    > 不同版本 `kubectl` 输出可能略有不同。
+
+- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/)：版本 >= 0.4.0
+- [net.ipv4.ip_forward](https://linuxconfig.org/how-to-turn-on-off-ip-forwarding-in-linux) 需要被设置为 1
+
+## 第 1 步: 通过 kind 部署 Kubernetes 集群
+
+首先，请确认 Docker 进程正常运行。然后你可以通过脚本命令快速启动一个本地的单点 Kubernetes 集群。
+
+1. Clone 代码：
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]:
```suggestion
1. Clone 官方提供的代码：
```

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+    ``` shell
+    git clone --depth=1 https://github.com/pingcap/tidb-operator && \
+    cd tidb-operator
+    ```
+
+2. 创建集群：
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]:
```suggestion
2. 运行脚本，在本地创建一个 Kubernetes 集群：
```

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+    ``` shell
+    hack/kind-cluster-build.sh
+    ```
+
+    > **注意：**
+    >
+    > 通过该脚本启动的 kind 集群默认有 6 个集群节点，kubernetes 版本默认为 v1.12.8，每个节点默认挂载数为 9。你可以通过启动参数去修改这些参数：
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]:
```suggestion
    > 通过该脚本启动的 Kubernetes 集群默认有 6 个节点，Kubernetes 版本默认为 v1.12.8，每个节点默认挂载数为 9。你可以通过启动参数去修改这些参数：
```

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 12]
```diff
@@ -0,0 +1,130 @@
...
+
+3. 等待集群创建完毕以后，我们切换 kube-config 文件来连接到本地 Kubernetes 集群:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    export KUBECONFIG="$(kind get kubeconfig-path)"
+    ```
+
+4. 查看本地 kind kubernetes 集群信息:
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]:
```suggestion
4. 查看该 kubernetes 集群信息：
```

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 13]
```diff
@@ -0,0 +1,130 @@
...
+    ```
+
+    输出如下类似信息:
+
+    ``` shell
+    Kubernetes master is running at https://127.0.0.1:50295
+    KubeDNS is running at https://127.0.0.1:50295/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+    ```
+
+5. 查看本地 kind kubernetes `storageClass`:
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]:
```suggestion
5. 查看该 Kubernetes 集群的 `storageClass`：
```

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 14]
```diff
@@ -0,0 +1,130 @@
...
+
+    输出如下类似信息:
+
+    ``` shell
+    NAME                 PROVISIONER                    AGE
+    local-storage        kubernetes.io/no-provisioner   7m50s
+    standard (default)   kubernetes.io/host-path        8m29s
+    ```
+
+## 第 2 步: 在 kind Kubernetes 集群上部署 TiDB Operator
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]:
```suggestion
## 第 2 步: 在 Kubernetes 集群上部署 TiDB Operator
```

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 15]
```diff
@@ -0,0 +1,130 @@
...
+
+参考[部署 TiDB Operator](/dev/tidb-in-kubernetes/deploy/tidb-operator.md#安装-tidb-operator)中的操作。
+
+## 第 3 步: 在 kind Kubernetes 集群中部署 TiDB 集群
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]:
```suggestion
## 第 3 步: 在 Kubernetes 集群中部署 TiDB 集群
```

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 16]
```diff
@@ -0,0 +1,130 @@
...
+
+参考[标准 Kubernetes 上的 TiDB 集群](/dev/tidb-in-kubernetes/deploy/general-kubernetes.md#部署-tidb-集群)中的操作。
+
+## 访问数据库和监控面板
+
+参考[查看监控面板](/dev/tidb-in-kubernetes/monitor/tidb-in-kubernetes.md#查看监控面板)中的操作。
+
+## 删除 TiDB 集群 与 kind Kubernetes 集群
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]:
```suggestion
## 删除 TiDB 集群 与 Kubernetes 集群
```

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 17]
```diff
@@ -0,0 +1,130 @@
...
+
+删除本地 TiDB 集群可参考[销毁 TiDB 集群](/dev/tidb-in-kubernetes/maintain/destroy-tidb-cluster.md#销毁-kubernetes-上的-tidb-集群)。
+
+通过下面命令删除 kind Kubernetes 集群:
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]:
```suggestion
通过下面命令删除该 Kubernetes 集群:
```

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 18]
```diff
@@ -0,0 +1,130 @@
...
+title: 使用 kind 在 Kubernetes 上部署 TiDB 集群
+summary: 使用 kind 在 Kubernetes 上部署 TiDB 集群。
+category: how-to
+---
+
+# 使用 kind 在 Kubernetes 上部署 TiDB 集群
+
+本文介绍了如何在个人电脑（Linux 或 MacOS）上采用 [kind](https://kind.sigs.k8s.io/) 方式在 Kubernetes 上部署 [TiDB Operator](https://github.com/pingcap/tidb-operator) 和 TiDB 集群。
+
+kind 通过 Docker 容器模拟出一个单点的 Kubernetes 集群。kind 的设计初衷是为了在本地进行 Kubernetes 集群的一致性测试，这意味着你可以使用 kind 模拟出你想要的 Kubernetes 版本集群。你可以在 [Docker hub](https://hub.docker.com/r/kindest/node/tags) 中找到你想要部署的 Kubernetes 版本。
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]: Why it's a '单点的 Kubernetes 集群'?
There are several nodes in the kind cluster, right?

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md (hunk 8 above)
Comment (Yisaer) [reply dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]: fix

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md (hunk 18 above)
Comment (Yisaer) [reply dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]: fix

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 19]
```diff
@@ -0,0 +1,130 @@
...
+    > **注意：**
+    >
+    > 不同版本 `kubectl` 输出可能略有不同。
+
+- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/)：版本 >= 0.4.0
+- [net.ipv4.ip_forward](https://linuxconfig.org/how-to-turn-on-off-ip-forwarding-in-linux) 需要被设置为 1
+
+## 第 1 步：通过 kind 部署 Kubernetes 集群
+
+首先，请确认 Docker 进程正常运行。然后你可以通过脚本命令快速启动一个本地的单点 Kubernetes 集群。
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]: another ‘单点’

Comment (Yisaer) [reply dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]: fix

**Pull request merged**